### DISCUSSION TITLE
How about the the nordtheme patches for st-flexipatch?

### DISCUSSION THREAD

**Discussion by zedxot** [bakkeby/st-flexipatch#49] [Ideas]
Hello, I'm a big fan of the flexipatch edition of every suckless program. It made my life easy and freed a lot of us from all the time we're spending to patch those program. I've been using flexipatch programs for a while now. Recently I was thinking about switching to nord theme from pywal. I noticed there is a st patch on suckless website called nordtheme. But when I browse through st-flexipatch patches I couln't find this particular patch there. I wonder if there's an way to include it or an easy way to make it work.

**Comment by bakkeby** [comment]
This is similar to
https://github.com/bakkeby/st-flexipatch/issues/17#issuecomment-832813172

There are several ways to add the nord theme:
   - apply the configuration settings from the patch manually (it is just configuration after all) or
  - use pywal to set the theme (and to restore the theme when starting a new terminal) or
  - use [theme.sh](https://github.com/lemnos/theme.sh) to set the theme or
  - use Xresources (and the associated patch) to control the colours for st

If you go with the first option and you tend to change theme once in a while then you may want to consider moving the colour settings to a separate file that is included into config.h, makes changing themes easier to manage. 

Using Xresources is a fairly common choice I think as you don't need to recompile st for the changes to come into effect. 

pywal and theme.sh are useful to try color schemes out, but needs extra stuff to make those colours appear on new instances of st.
